stats.c

```diff
@@ -1612,7 +1621,18 @@ void output_stats(FILE *to, stats_t *stats, int sparse)
                 100.*(acgtno_count_1st->t + acgtno_count_2nd->t)/acgt_sum,
                 100.*(acgtno_count_1st->n + acgtno_count_2nd->n)/acgt_sum,
                 100.*(acgtno_count_1st->other + acgtno_count_2nd->other)/acgt_sum);
-
+    }
+    fprintf(to, "# ACGT reverse complemented content per cycle. Use `grep ^GCT | cut -f 2-` to extract this part. The columns are: cycle; A,C,G,T base counts as a percentage of all A/C/G/T bases [%%]\n");
```

Comment:
`ACGT reverse complemented content per cycle` implies the data is always reverse complemented.  Maybe something like `ACGT content per cycle, read oriented` or `Read-oriented per-cycle ACGT content` would be better?